addresshelper/helper.go

```diff
@@ -17,25 +19,23 @@ func PkScript(address string) ([]byte, error) {
 }
 
 func DecodeForNetwork(address string, params *chaincfg.Params) (dcrutil.Address, error) {
-	addr, err := dcrutil.DecodeAddress(address)
+	addr, err := dcrutil.DecodeAddress(address, params)
```

Comment:
Can simply `return dcrutil.DecodeAddress(address, params)`. This `DecodeForNetwork` function may actually no longer be needed as `dcrutil.DecodeAddress(address, params)` can instead be used directly.